.github/workflows/integrate.yaml

```diff
@@ -24,6 +24,10 @@ jobs:
           - "test_scale_core"
           - "test_scale_tools"
           - "test_upgrade"
+          - "test_cmd_with_native_ssh"
+          - "test_cmd_no_cdc_with_native_ssh"
```

Comment:
This is not necessary, it's testing the logic of cluster deploying, not related to the SSH implementation.